keras_hub/src/tests/test_case.py

```diff
@@ -720,6 +720,47 @@ def compare(actual, expected):
             output = ops.argmax(output, axis=-1)
             self.assertAllEqual(output, expected_labels)
 
+    def run_positions_test(
```

Comment:
do we think this is generic and will extend beyond gemma? if so ok to leave here. if not I might park this directly in the gemma tests.